spanner/changestreams/src/main/java/com/example/spanner/changestreams/Utils.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner.changestreams;
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.spanner.admin.database.v1.UpdateDatabaseDdlMetadata;
+import java.util.List;
+
+public class Utils {
+  public static void printUsage(String className) {
+    System.err.println("Usage: " + className + " "
+        + "<instance_id> "
+        + "<database_id> "
+        + "<prefix>");
+  }
+
+  public static void updateDdl(String instanceId,
+                               String databaseId,
+                               List<String> statements) throws Exception {
+    Spanner spanner = SpannerOptions.newBuilder().build().getService();
```

Comment:
Could you move this into the constructor and make the methods non-static?

If you do so, make sure to give the constructor the project id and then you can construct the SpannerOptions with that as well:

```
SpannerOptions.newBuilder().setProject(projectId).build().getService();
```

You can close spanner afterwards in another method (like `close()`).

This is useful, because if you have not set a project id in your gcloud configuration, we would still use the one that was given through the parameters.

Reply:
> If you do so, make sure to give the constructor the project id and then you can construct the SpannerOptions with that as well:

Are you suggesting that we need the user to input project id as well?

> You can close spanner afterwards in another method (like close()).
You mean keep the Spanner object as a member of Utils, and provide a separate method just for closing Spanner object? When does the caller calls the close method though? At the end of main method?

Reply:
I am suggesting that - if the project, instance and database are always the same for the lifecycle of the `Utils` class - we could create a Utils constructor that receives those (or at least the project, which should not change). Then you can build the options in the constructor as well and get the service. Something like:


```java
public class Utils {
  public Utils(String projectId, String instanceId, String databaseId) {
    final SpannerOptions options = SpannerOptions.newBuilder().setProject(projectId).build();
    final DatabaseId id = DatabaseId.of(projectId, instanceId, databaseId);
    this.spanner = options.getService();
    this.databaseClient = spanner.getDatabaseClient(id); // this only works if the database is already created (I think that is the case)
    this.databaseAdminClient = spanner.getDatabaseAdminClient();
  }

  public void close() {
    spanner.close();
  }
}
```

Reply:
Thanks, removed the util since we do everything in one file.